vividus-extension-selenium/src/main/java/org/vividus/ui/variable/AbstractSearchContextRectangleDynamicVariable.java

```diff
@@ -20,24 +20,41 @@
 
 import org.openqa.selenium.Rectangle;
 import org.openqa.selenium.WebElement;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 import org.vividus.ui.context.UiContext;
 import org.vividus.variable.DynamicVariable;
 import org.vividus.variable.DynamicVariableCalculationResult;
 
+@Deprecated(forRemoval = true, since = "0.6.6")
```

Comment:
I guess it's needed to update documentation as well in order to indicate which variables are deprecated